config/sync/views.view.stanford_events.yml

```diff
@@ -5261,7 +5261,7 @@ display:
           exclude: 0
           alter:
             alter_text: 1
-            text: "<a class=\"su-link{% if su_event_source %} su-link--external{% endif %}\" href=\"{% if su_event_source %}{{ su_event_source }}{% else %}{{ view_node }}{% endif %}\">\r\n  {{ title }}\r\n</a>"
+            text: "<h3>\r\n  <a class=\"su-link{% if su_event_source %} su-link--external{% endif %}\" href=\"{% if su_event_source %}{{ su_event_source }}{% else %}{{ view_node }}{% endif %}\">\r\n  {{ title }}\r\n  </a>\r\n</h3>"
```

Comment:
This change is unnecessary. You removed the `h3` from one place, to add it in another. Please revert this change. You should only be making changes to the subheadline field, not the title field.

Reply:
Do you see the screenshot and summary text I added at the top of this PR? When I spin up a new instance of Stanford Profile, it's not rending the title correctly unless I update it this way.

![image](https://github.com/SU-SWS/stanford_profile/assets/49299083/8e4ec286-128d-42ec-93f6-c14efdc257a5)

Reply:
If you provide steps how to reproduce the issue, I'd be able to review. But at this time, the title field has the same markup.

Reply:
1. Open a new gitpod workspace for https://github.com/SU-SWS/stanford_profile
2. Refer to Step 2 under `Setup tasks and/or behavior to test` and confirm it looks incorrect

Reply:
I fixed it myself. see the commit: https://github.com/SU-SWS/stanford_profile/pull/783/commits/b9671b58f30aa47b6f851f989bbc3ab2c64b8dd8